Kitodo/src/test/java/org/kitodo/production/services/command/ImportProcessesIT.java

```diff
@@ -0,0 +1,330 @@
+import org.kitodo.data.database.beans.User;
+import org.kitodo.production.services.ServiceManager;
+
+public class ImportProcessesIT {
+    private static final Path ERRORS_DIR_PATH = Paths.get("src/test/resources/errors");
+
+    @BeforeClass
+    public static void prepareDatabase() throws Exception {
+        MockDatabase.startNode();
+        MockDatabase.insertProcessesFull();
+
+        // add ruleset
+        Ruleset ruleset = new Ruleset();
+        ruleset.setTitle("ImportProcessesIT");
+        ruleset.setFile("ImportProcessesIT.xml");
+        ruleset.setOrderMetadataByRuleset(true);
+        Client clientOne = ServiceManager.getClientService().getById(1);
+        ruleset.setClient(clientOne);
+        ServiceManager.getRulesetService().save(ruleset); // will get ID 5
+
+        Task task = new Task();
+        task.getRoles().add(ServiceManager.getRoleService().getById(1));
+        ServiceManager.getTaskService().save(task); // will get ID 14
+
+        Template template = new Template();
+        template.setTitle("Import processes template");
+        LocalDate localDate = LocalDate.of(2023, 8, 9);
+        template.setCreationDate(Date.from(localDate.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant()));
+        template.setClient(clientOne);
+        template.setDocket(ServiceManager.getDocketService().getById(1));
+        template.getProjects().add(ServiceManager.getProjectService().getById(1));
+        template.setRuleset(ServiceManager.getRulesetService().getById(5)); // ruleset added above
+        ServiceManager.getTemplateService().save(template, true); // will get ID 5
+
+        task = ServiceManager.getTaskService().getById(14);
+        task.setTemplate(ServiceManager.getTemplateService().getById(5));
+        template.getTasks().add(ServiceManager.getTaskService().getById(14));
+        ServiceManager.getTemplateService().save(template, true);
+        ServiceManager.getTaskService().save(task);
+        
+        Folder local = ServiceManager.getFolderService().getById(6);
+        local.setMimeType("image/jpeg");
+        ServiceManager.getFolderService().saveToDatabase(local);
+
+        User userOne = ServiceManager.getUserService().getById(1);
+        SecurityTestUtils.addUserDataToSecurityContext(userOne, 1);
+    }
+
+    @BeforeClass
+    public static void setScriptPermission() throws Exception {
+        if (!SystemUtils.IS_OS_WINDOWS) {
+            ExecutionPermission
+                    .setExecutePermission(new File(ConfigCore.getParameter(ParameterCore.SCRIPT_CREATE_DIR_META)));
+        }
+    }
+
+    @BeforeClass
+    public static void createOutputDirectories() throws Exception {
+        Files.createDirectories(ERRORS_DIR_PATH);
+    }
+
+    @AfterClass
+    public static void deleteCreatedFiles() throws Exception {
+        TreeDeleter.deltree(Paths.get("src/test/resources/metadata/4/images"));
+        TreeDeleter.deltree(Paths.get("src/test/resources/metadata/5/images"));
```

Comment:
Please do not use static process IDs. Removing those static IDs was the whole point of #5876

Reply:
The test tests importing processes from files. Not the test, the piece of code under test does create these files, because it is its job. The test code here cleans up. I don’t know how to do that without using the IDs?